articles/multifactor-authentication/administrator/guardian-for-select-users.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Guardian for Select Users
+---
+## Apply Guardian only to certain users
+
+Once you have enabled either MFA option, you will be presented with the **Customize MFA** code snippet that allows advanced configuration of Guardian's behavior via [Rules](/rules). One option is to apply Guardian authentication only to a subset of your users.
+
+By default, Auth0 enables Guardian for all accounts.
+
+```js
+function (user, context, callback) {
+
+  // Apply Guardian only for users that have user_metadata.use_mfa === true
+  if (user.user_metadata && user.user_metadata.use_mfa){
+      context.multifactor = {
+        provider: 'guardian', //required
+      };
+  }
+
+  callback(null, user, context);
+}
+```
+
+If you choose to selectively apply multifactor authentication, you simply set the appropriate `clientID` values, and the code will be executed as part of a [Rule](/rules) whenever a user logs in.
```

Comment:
I think this does not belong to this type of customization (there is no clientId in the rule). Also maybe we should say that to enable multifactor for an user with this rule you should set user_metadata.use_mfa to true using the api2